components/Itemlist.js

```diff
@@ -0,0 +1,74 @@
+import { shoes } from "../public/shoes";
+import styled from "styled-components";
+
+export default function Itemlist() {
+  return (
+    <>
+      {shoes.map((shoe) => (
+        <StyledItemList>
+          <StyledImage src={shoe.img} width="50%" height="70%" />
+          <StyledListContainer>
```

Comment:
You are using a 'ul' here - with only one ListItem. 
you might not need the 'ul ' here.